src/components/RelatedAssets/RelatedAssets.tsx

```diff
@@ -20,6 +20,10 @@ export const RelatedAssets: React.FC<RelatedAssetsProps> = ({ assetId }) => {
     selectRelatedAssetIds(state, relatedAssetIdsFilter),
```

Comment:
Gut feel is this could come and bit us in a few more places if we ever use this selector somewhere else (which we currently don't) - do we want to do that at selector-level?

Reply:
For sure - moved to selector level in https://github.com/shapeshift/web/pull/7626/commits/48faacce4c27887748dcd3a72a3acf5789c560a2.

Much cleaner 🤝